.travis.yml

```diff
@@ -19,15 +19,13 @@ jobs:
 
   - stage: release
     script:
-      - echo "$PGP_SECRET" | base64 --decode | gpg --import
-      - ./mill -j $(nproc) __.jvm[__].publish --sonatypeCreds "$SONATYPE_USERNAME:$SONATYPE_PASSWORD" --release true --gpgPassphrase "$PGP_PASSPHRASE"
-
-  - stage: release
-    script:
-      - echo "$PGP_SECRET" | base64 --decode | gpg --import
-      - ./mill -j $(nproc) __.js[__].publish --sonatypeCreds "$SONATYPE_USERNAME:$SONATYPE_PASSWORD" --release true --gpgPassphrase "$PGP_PASSPHRASE"
-
-  - stage: release
-    script:
-      - echo "$PGP_SECRET" | base64 --decode | gpg --import
-      - ./mill -j $(nproc) __.native[__].publish --sonatypeCreds "$SONATYPE_USERNAME:$SONATYPE_PASSWORD" --release true --gpgPassphrase "$PGP_PASSPHRASE"
+      - echo $GPG_PRIVATE_KEY_B64 | base64 --decode > gpg_key
+      - gpg --import gpg_key 
+      - rm gpg_key
+      - ./mill mill.scalalib.PublishModule/publishAll \
+        --sonatypeCreds lihaoyi:$SONATYPE_PASSWORD \
+        --gpgArgs --passphrase=$GPG_PASSWORD,--batch,--yes,-a,-b \
+        --publishArtifacts __.publishArtifacts \
+        --readTimeout 600000 \
+        --release true \
+        --signed true
```

Comment:
This script was taken from mill: https://github.com/lihaoyi/mill/blob/master/ci/release-maven.sh